docker/compose.sh

```diff
@@ -10,12 +10,22 @@ if [ "$1" == "init" ]; then
     docker rm file-queue-helper
 
     docker-compose -p instant -f "$composeFilePath"/docker-compose.yml -f "$composeFilePath"/docker-compose.dev.yml -f "$composeFilePath"/importer/docker-compose.config.yml up -d
+
+    echo "Sleep 90 - Give importers time to complete"
+    sleep 90
+
+    # docker rm covid19-surveillance-openhim-config-importer covid19-surveillance-config-importer dhis2-tracker-populator-config-importer hapi-fhir-config-importer dhis2-config-importer
```

Comment:
The logic for removing the containers should be uncommented

Reply:
Lol, good catch 🙈